requirements.txt

```diff
@@ -39,3 +39,4 @@ whitenoise==5.0.1
 django-constance[database]==2.7.0
 invoke==1.4.1
 werkzeug==1.0.1
+celery==5.0.1
```

Comment:
This should be applied as a `hotfix` ASAP, `master` is broken in #120

I'm gonna patch it, k8s deployments are broken 😨